packages/thumbnail/src/thumbnail.css

```diff
@@ -12,6 +12,6 @@ governing permissions and limitations under the License.
 
 @import './spectrum-thumbnail.css';
 
-::slotted(:not(img)) {
+/* ::slotted(:not(img)) {
```

Comment:
I think we will want this back after the internal structure of the element is correct to the updated implementation.

Reply:
it has been put back!